mlir/lib/Dialect/Transform/IR/TransformOps.cpp

```diff
@@ -1463,6 +1526,7 @@ void transform::ForeachOp::getSuccessorRegions(
 
 OperandRange
 transform::ForeachOp::getEntrySuccessorOperands(RegionBranchPoint point) {
+  // TODO: figure out how to update the comment & impl if necessary
```

Comment:
This logic still applies.

Reply:
Thanks. Could you check that my updated comment here is accurate/relevant?